extensions/wikia/AdEngine/js/context/adContext.js

```diff
@@ -86,10 +86,6 @@ define('ext.wikia.adEngine.adContext', [
 				geo.isProperGeo(instantGlobals.wgAdDriverSourcePointDetectionMobileCountries));
 		}
 
-		// SourcePoint recovery integration (set in AdEngine2ContextService based on wgEnableUsingSourcePointProxyForCSS)
-		if (isUrlParamSet('sourcepointrecovery')) {
```

Comment:
Does it make sense to be consistent and remove `sourcepointdetection` URL param as well? Please updated docs too ;)